Mentioned update to spryker/twig 3.15.2 (#1627)
spryker/spryker-docs: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

You cannot successfully deploy a project on Spryker Cloud Commerce OS unless you do the following:

- Upgrade your project's code to match the latest demoshop release, or at minimum, upgrade to a release that fully supports the Docker SDK.

> - Make sure you have updated spryker/twig to at least version 3.15.2 as this version has important stability improvements over version 3.15.1

Reviewer: ```suggestion
- Update spryker/twig to version 3.15.2 or higher because this version and higher have important stability improvements over version 3.15.1.
```
